### PR TITLE
LPD-23331 Make sure workflow info is displayed when task is approved

### PR DESCRIPTION
**Pull request by noornajj**
https://liferay.atlassian.net/browse/LPD-23331

### CODE REVIEW

**Comment by noornajj** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
@noornajj this looks good to me, but we need a test for this. Can you add this to our playwright test?

**Comment by noornajj** [comment]
@pei-jung Do you want me to add it in the same ticket as well?

**Comment by pei-jung** [comment]
Yes, please add it in this ticket

**Comment by noornajj** [comment]
Hi @pei-jung 
I have included the playwright test

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
@noornajj can you fix the merge conflict?

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingRenderView.js [hunk 1]
```diff
@@ -1337,7 +1337,7 @@ export default function ChangeTrackingRenderView({
 			);
 		}
 
-		if (workflowStatus && showWorkflow) {
+		if (workflowStatus !== null && showWorkflow) {
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingRenderView.js:1340]: If parenthesis is allowed, let's add it around `workflowStatus !== null` so it's easier to read.

Comment (pei-jung) [reply modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingRenderView.js:1340]: Ok, let's leave it as is then.

**Comment by noornajj** [comment]
Conflict resolved

**Comment by pei-jung** [comment]
ci:forward

**Comment by noornajj** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:forward

**Comment by pei-jung** [comment]
@noornajj failures look unrelated. Can you manually forward to BChan?

**Pull request closed without merging**